assignments/arrays.js

```diff
@@ -63,32 +63,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+const getid = inventory.find(inventory => inventory.id === 33);
+// console.log(getid.car_year);
+console.log('Car 33 is a ' + getid.car_year + " " + getid.car_make + " " + getid.car_model);
```

Comment:
While functional, please get used to using string interpolation.

Car 33 is a ${getid.car_year} ${getid.car_make} ${getid.car_model}

It's much faster to write this way and is the industry standard.